src/use_file.rs

```diff
@@ -29,28 +31,31 @@ const FILE_PATH: &str = "/dev/urandom";
 const FILE_PATH: &str = "/dev/random";
 
 pub fn getrandom_inner(dest: &mut [u8]) -> Result<(), Error> {
-    lazy_static! {
-        static ref FILE: Result<File, Error> = init_file();
-    }
-    let mut f = FILE.as_ref()?;
+    static FD: LazyFd = LazyFd::new();
+    let fd = FD.init(init_file).ok_or(Error::UNKNOWN)?;
```

Comment:
I think you can use `last_os_error` and convert it into a code. In future full migration from `std` it can be replaced with `errno`.

Reply:
Done